config/testdata/example_config.yaml

```
@@ -41,9 +41,16 @@ provider:
   policyEtagFlushDur: 24h
   roletoken:
       enable: true
-  access_tokens:
-    -
-      enable: true
-      verify_cert_thumbprint: true
-      cert_backdate_dur: 1h
-      cert_offset_dur: 1h
+  access_token:
+    enable: true
+    verify_cert_thumbprint: true
+    verify_token_client_id: true
```

```suggestion
    verify_token_client_id: false
```
let's follow authorizer's default